app/common/ValueFormatter.ts

```diff
@@ -79,19 +79,25 @@ export class BaseFormatter {
    * Formats using this.format() if a value is of the right type for this formatter, or using
    * AnyFormatter otherwise. This method the recommended API. There is no need to override it.
    */
-  public formatAny(value: any): string {
-    return this.isRightType(value) ? this.format(value) : formatUnknown(value);
+  public formatAny(value: any, translate?: Function): string {
+    return this.isRightType(value) ? this.format(value, translate) : formatUnknown(value);
   }
 
   /**
    * Formats a value that matches the type of this formatter. This should be overridden by derived
    * classes to handle values in formatter-specific ways.
    */
-  protected format(value: any): string {
+  protected format(value: any, translate?: Function): string {
     return String(value);
   }
 }
 
+export class BoolFormatter extends BaseFormatter {
+  public format(value: boolean, translate?: (val: string) => string): string {
+    return translate ? translate(String(value)) : String(value);
```

Comment:
```suggestion
  public format(value: boolean | 0 | 1, translate?: (val: string) => string): string {
    if (typeof value === 'boolean' && translate) {
      return translate(String(value));
    }

    return super.format(value, translate);
```

What do you think of this approach? It should remove the need for creating translation keys for 0/1 (which aren't displayed, unlike true/false).

Reply:
It's fine for me.